src/assets/release-notes-1.17.json

```diff
@@ -0,0 +1,2629 @@
+{
```

Comment:
I don't think we need these here since they're already part of the master branch :)